app.js

```diff
@@ -1,46 +1,138 @@
 "use strict";
 
-const express = require("express"); // web application framework
+const express = require("express");
+const bodyParser = require("body-parser");
 const app = express();
 const child_process = require("child_process");
+const fs = require("fs");
+const pngToJpeg = require("png-to-jpeg");
 
-// set server to use preconfigured port or 8000
-app.set("port", (process.env.PORT || 8000));
+app.set("port", (process.env.PORT || 8000)); // set server to use preconfigured port or 8000
+app.use(bodyParser.json({limit: "50mb"}));
+app.use(bodyParser.urlencoded({limit: "50mb", extended: true, parameterLimit: 50000}));
+
+/**
+ * Writes a base64 string for a JPEG file to disk as a JPEG file
+ * @param {string} base64String - base64 string of JPEG file
+ */
+function writeJpegToDisk(base64String) {
+    let base64Data = base64String.replace(/^data:image\/jpeg;base64,/, ""); // stripping for decoding
+    // writing file to disk as "out.jpeg"
+    fs.writeFile("out.jpeg", base64Data, "base64", function (error) {
+        if (error !== null) {
+            console.log(error);
+        }
```

Comment:
You should probably handle the error somehow. Just letting it silently fail is asking for trouble